### ISSUE TITLE
Correct missing outline for logo in the header

### ISSUE THREAD

**Issue by maxliqd** [liqd/digitalstrategie#608]
**Essential Information**
**URL:** gemeinsamdigital.berlin.de
**device & browser:**
**operating system (incl. version):**
**behaviour:**
Operation via keyboard is possible.

The focus at the logo "Gemeinsam Digital" is not recognizable, a border is missing.
**expected behaviour:**
All relevant elements can be reached via the keyboard. The sequence using the TAB key is as expected. The focus is not caught in one form, so that other areas are not reached. The highlighting of the currently active element is always recognizable.

**Additional information if relevant**
**user role:**
**important screensize:**
**Comment/Question:**

**Screenshot if design related**:
Since issues are public, make sure to not display personal or secret data.

**Comment by philli-m** [comment]
@maxliqd can't reproduce, which browser and version?

**Comment by maxliqd** [comment]
@philli-m it does not work on firefox for the GD:B Logo and on safari it does not work for Berlin and GD:B Logo.

**Comment by maxliqd** [comment]
And neither on chrome for the GD:B logo

**Comment by maxliqd** [comment]
<img width="439" alt="image" src="https://github.com/liqd/digitalstrategie/assets/94360651/810d520b-2e59-498c-ab14-5a36000fbfe1">

And this is how I understand it should look like for the both logos whenever you tab to them

**Comment by philli-m** [comment]
@maxliqd can you tell me which browser versions and OS version your mac is on? outline is visible on all browsers for me

**Comment by maxliqd** [comment]
Chrome Version 113.0.5672.126 (Offizieller Build) (arm64)
Firefox Version 113.0.2 (64-Bit)
Safari Version 16.2 (18614.3.7.1.5)
macOS Ventura Version 13.1

**Comment by maxliqd** [comment]
Tested on browser stack windows 11, firefox 113: 

https://github.com/liqd/digitalstrategie/assets/94360651/7e0b92d1-907e-4cbc-842b-d05d604a1c5a

**Comment by philli-m** [comment]
@maxliqd closing as works on dev?
